IBPSA/Fluid/Actuators/Valves/TwoWayPolynomial.mo

```diff
@@ -0,0 +1,43 @@
+within IBPSA.Fluid.Actuators.Valves;
+model TwoWayPolynomial "Two way valve with polynomial characteristic"
+  extends IBPSA.Fluid.Actuators.BaseClasses.PartialTwoWayValveKv(
+    phi=l + pol_y*(1 - l));
+
+  parameter Real[:] c "Polynomial coefficients, starting with fixed offset";
+
+protected
+  Real pol_y = sum(c.*{y_actual^i for i in 0:size(c,1)-1}) "Polynomial of valve control signal";
+
+equation
+  // We provide some slack on these bounds to avoid the assert failing when
+  // small integration errors are made, e.g. in the filter.
+  assert((pol_y>=-0.2 and pol_y <=1.2),
+    "The provided polynomial coefficients lead 
+    to a valve opening larger than 1 or smaller than 0");
+
+annotation (
+defaultComponentName="val",
+Documentation(info="<html>
+<p>
+Two way valve with polynomial opening characteristic.
+The polynomial coefficients are defined using parameter <code>c</code>.
+The elements of c are coefficients for increasing powers of y,
+starting with the power 0, which corresponds to a fixed offset.
+</p>
+<p>
```

Comment:
Please add why a user would choose this valve to avoid users being confused what to select. Maybe this should be in the user guide as well?

Reply:
I extended the description. There is currently no overview of the two/three way valve models in the user guide, so there was no obvious location where to mention the new valve model. Is this something we'd like to add?

Reply:
@mwetter : please see my reply :)